### DISCUSSION TITLE
新加坡試行「巡邏機器人」 偵測不良社會行為，引侵犯隱私疑慮

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#390] [科技新知]
大家都知道新加坡是個乾淨且嚴格的國家，也是世界上犯罪率最低的國家之一。
「政府最新的監控裝置是附有輪子的「Xavier機器人」，機器人身上配備7個鏡頭，可偵測「不良社會行為」，並向民眾發出警告。
所謂「社會不良行為」包括在非吸菸區吸菸、亂停腳踏車，以及違反COVID-19（2019冠狀病毒疾病）防疫社交距離規定等。」

但是他們使用AI機器人偵測不良行為的方式，可能大幅犧牲人們隱私權，人們對於自己的個人資料將如何被利用無從而知。
現今人們似乎越來越不在乎或無感於自己隱私權的剝奪了，你在乎嗎?
資料來源：https://www.bnext.com.tw/article/65461/xavier-robot-singapore-issue?

**Comment by YuHengChen54** [comment]
新加坡這部分法律真的很嚴格，太可怕了....

**Comment by BingRong1217** [comment]
巡邏機器人可以有效的減少警力

**Comment by Ben6uo** [comment]
但新加坡原本治安就很好了 感覺必要性不是很大

**Comment by brian431** [comment]
對於個人資料，我個人除了一些比較敏感的資料以外，其他的並不怎麼在意，儘管拿去用。
